### PR TITLE
[trade republic] Add status code to ignore some type of documents

### LINKED ISSUE #451: Implement new status code for known and not wanted to parse documents
- Status code: `7`
- There are many documents the user reported but we won't to parse (eg. Kosteninformation from Trade Republic, stock splits, ...).
- We need to detect these documents and return the status code. 
- ~~I will update the import-submit-tool to filter these documents and prevent wontfix issues.~~

### PR DESCRIPTION
**Pull request by dredav**
- Implement status codes `7` and `8`
- The following trade republic documents will return status code `8`: `KOSTENINFORMATION `, `SPARPLANAUSFÜHRUNG FEHLGESCHLAGEN`, `SPLIT`, `REVERSE SPLIT`
- Fix #451 (and all related issues)
- Add test cases for all the above documents
- Other implementations will soon be adjusted

### CODE REVIEW

File: src/brokers/traderepublic.js [hunk 1]
```diff
@@ -209,14 +209,30 @@ const isOverviewStatement = content =>
...
     isSell(content) ||
     isDividend(content) ||
-    isOverviewStatement(content));
+    isOverviewStatement(content)
+  );
+};
+
+export const canParsePage = (content, extension) =>
+  extension === 'pdf' &&
+  content.some(line => line.includes('TRADE REPUBLIC BANK GMBH'));
```
Comment (SirGibihm) [inline src/brokers/traderepublic.js:236]: I am unsure why we require satus code 7 . Couldn't we return false for `canParsePage()` if `(isSupportedDocument(firstPdfPage) || detectedButIgnoredDocument(firstPdfPage))`?

File: src/brokers/traderepublic.js [hunk 2]
```diff
@@ -209,14 +209,30 @@ const isOverviewStatement = content =>
...
-  content.some(line => line.includes('TRADE REPUBLIC BANK GMBH')) &&
-  (isBuySingle(content) ||
+const detectedButIgnoredDocument = content => {
+  return (
+    // When the document contains one of the following lines, we want to ignore these document.
+    content.some(line => line.includes('KOSTENINFORMATION')) ||
+    // Only exclude these document, when a line contains one of the following:
+    content.some(line => line === 'SPARPLANAUSFÜHRUNG FEHLGESCHLAGEN') ||
+    content.some(line => line === 'SPLIT') ||
+    content.some(line => line === 'REVERSE SPLIT')
```
Comment (SirGibihm) [inline src/brokers/traderepublic.js:219]: Easier and more resource efificient I think: `content.includes('SPLIT')`. Also two comments explaining the same thing; am I right?

**Review by SirGibihm — CHANGES_REQUESTED** [review]

File: src/brokers/traderepublic.js (hunk 2 above)
Comment (dredav) [reply src/brokers/traderepublic.js]: Yes you are!

File: src/brokers/traderepublic.js (hunk 1 above)
Comment (dredav) [reply src/brokers/traderepublic.js:200]: Good idea. We can remove one of the status code and it's easier to understand.

**Comment by dredav** [comment]
@SirGibihm Changes are in.

**Review by SirGibihm — APPROVED** [review]

**Pull request merged**